flink-connectors/flink-connector-elasticsearch-base/src/main/java/org/apache/flink/streaming/connectors/elasticsearch/ElasticsearchApiCallBridge.java

```diff
@@ -73,6 +73,16 @@
     @Nullable
     Throwable extractFailureCauseFromBulkItemResponse(BulkItemResponse bulkItemResponse);
 
+    /**
+     * Sets the bulk flush interval, in milliseconds on the provided {@link BulkProcessor.Builder}.
+     * The builder will be later on used to instantiate the actual {@link BulkProcessor}.
+     *
+     * @param builder the {@link BulkProcessor.Builder} to configure.
+     * @param flushIntervalMillis the flush interval in milliseconds.
+     */
+    void configureBulkProcessorFlushInterval(
```

Comment:
Why do we add this configuration option to the old connector?

Reply:
Bumping to 7.15.2 means that some imported classes changed their modules which breaks the current connector code. To keep the old connector working I had to move this to the version agnostic bridge interface